builddefs/build_keyboard.mk

```diff
@@ -176,6 +176,24 @@ ifeq ($(strip $(CONVERT_TO_PROTON_C)), yes)
     include platforms/chibios/boards/QMK_PROTON_C/convert_to_proton_c.mk
 endif
 
+ifeq ($(strip $(STMC)), yes)
+  CONVERT_TO_STEMCELL=yes
+endif
+
+ifeq ($(strip $(STMC_US)), yes)
+  CONVERT_TO_STEMCELL_UART_SWAP=yes
+  OPT_DEFS += -DCONVERT_TO_STEMCELL_UART_SWAP
+endif
+
+ifeq ($(strip $(STMC_IS)), yes)
+  CONVERT_TO_STEMCELL_I2C_SWAP=yes
+  OPT_DEFS += -DCONVERT_TO_STEMCELL_I2C_SWAP
+endif
```

Comment:
Can these move to `convert_to_stemcell.mk` given that they're not relevant otherwise?